app/code/Magento/NewRelicReporting/etc/csp_whitelist.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<csp_whitelist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Csp:etc/csp_whitelist.xsd">
+    <policies>
+        <policy id="script-src">
+            <values>
+              <value id="newrelic" type="host">*.newrelic.com</value>
+              <value id="newrelic-data" type="host">*.nr-data.com</value>
+              <value id="newrelic-nerd" type="host">*.nr-ext.net</value>
+              <value id="newrelic-assets" type="host">*.nr-assets.net</value>
+            </values>
+        </policy>
+        <policy id="connect-src">
+            <values>
+                <value id="newrelic-data" type="host">*.newrelic.com</value>
```

Comment:
the id must be unique for all values in a policy, otherwise only the last entry for a given id will be rendered => change the id to "newrelic"

Reply:
Change done. Thanks!

Reply:
Still unsure why the status is not updated on this MR: https://github.com/Serfe/magento2/commit/346e59a16ab4210a2a3df666305de8d2b8bb08f2